linter/rules/primary_key_is_not_first_dimension_in_view.py

```diff
@@ -0,0 +1,17 @@
+from linter.rule import Rule
+from typing import Any, Tuple
+
+
+class PrimaryKeyIsNotFirstDimensionInView(Rule):
```

Comment:
Again the rule name should be what we _want_ to enforce. In this case, we _want_ the primary key to be the first dimension:

```
PrimaryKeyIsFirstDimensionInView
```

The reason we have negative rule names for some others is because we are enforcing the negative case:

```
YesnoFieldLabelShouldNotContainYesno
```

We should also probably roll with either "is" or "should" for both cases, but that's a separate topic, and I can standardize that after we finish all the rules.